addon-api/ping-server.js

```diff
@@ -4,13 +4,14 @@
  * http://mozilla.org/MPL/2.0/.
 */
 
-const prefs = require('sdk/simple-prefs').prefs;
+const config = require('./package.json').CONFIG;
```

Comment:
We should have a way for an individual projects to override this config - maybe these end up being methods on an object initialized with the config? (Or could pass config to each function, but that feels messy.) An experiment addon will probably want to change these with prefs, or pull the values from the Idea Town addon prefs.

Should also ship this to NPM with `ideatown.firefox.com` production config by default (even though it doesn't exist yet)

Maybe this plays into #175 for settings in dev / prod switching?